server/world/mcdb/provider.go

```diff
@@ -28,6 +28,13 @@ type Provider struct {
 // chunkVersion is the current version of chunks.
 const chunkVersion = 27
 
+var (
+	// lastOpenedWithVersion is the current last opened with version, required by the latest Minecraft data provider.
+	lastOpenedWithVersion = []int32{1, 18, 0, 2, 0}
+	// minimumCompatibleClientVersion is the minimum compatible client version, required by the latest Minecraft data provider.
+	minimumCompatibleClientVersion = []int32{1, 18, 0, 0, 0}
```

Comment:
Do you reckon we could base these on the protocol version? I'm not a huge fan of having to update these everytime. I suppose the minimum might not be easy to do but with the lastOpenedWithVersion that should be possible(?).

Reply:
Also are all 3 of NetworkVersion, LastOpenedWithVersion and MinimumCompatibleClientVersion needed for it to work?

Reply:
I haven't tried each one of them separately, but honestly better to be safe then sorry. I can look into basing these off of protocol.CurrentVersion though.

Reply:
That would be a good improvement.

Reply:
Any update on this? Do you reckon we could resolve this and get this merged by the end of the month?

Reply:
Updated.